### PR TITLE
document: add CHANGELOG

### PR DESCRIPTION
**Pull request by sfwn**
#### What type of this PR

/kind document

#### What this PR does / why we need it:

add CHANGELOG for v1.0.0 & v1.0.1.

preview: https://github.com/sfwn/erda/tree/feature/add-changelog

#### Need cherry-pick to release versions?

`/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by hoozecn** [comment]
@sfwn Please draft your pr if it's not ready

**Comment by sfwn** [comment]
> @sfwn Please draft your pr if it's not ready yet or reviewers will receive too many notifications. Thx



> @sfwn Please draft your pr if it's not ready yet or reviewers will receive too many notifications. Thx

ready now.

File: CHANGELOG/CHANGELOG-1.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Erda Changelog
+
+- [v1.0.1](#v101-2021-07-08)
+- [v1.0.0](#v100-2021-06-09)
+
+# v1.0.1 [2021-07-08]
+
+## New features
+
+- Support individual develop to create organization if he/she doesn't belong to any organizations ([#592](https://github.com/erda-project/erda/pull/592))
```
[the remark this answers is not in the document]
Comment (sfwn) [reply CHANGELOG/CHANGELOG-1.0.md:10]: :P

**Review by Dreamica — CHANGES_REQUESTED** [review]
New Features

- Support individuals to create organization if he/she doesn't belong to any organizations.
- Support scaling applications without restarting the existing instances.

Fixed Issues

- Fix the issue that custom stages were created with null value.
- Keep the creator, assignee, create time and man hour unchanged when issue type is switched.
- Fix the logic error of job deletion under the specified namespace.
- Add guest permissions for dashboard and ticket.
- Reset flags before loop for wait step when timed out.

**Pull request merged**